src/components/PortalWrapper/PortalWrapper.jsx

```diff
@@ -299,20 +314,22 @@ const PortalWrapper = (options = defaultOptions) => ComposedComponent => {
             timeout={timeout}
             {...rest}
           >
-            <Content manager={this._MrManager} id={this._portalWrapperId}>
-              <ComposedComponent
-                className={className}
-                openPortal={openPortal}
-                closePortal={handleOnClose}
-                onClose={onClose}
-                portalIsOpen={portalIsOpen}
-                portalIsMounted={portalIsOpen}
-                forceClosePortal={this.forceClosePortal}
-                trigger={trigger}
-                zIndex={zIndex}
-                {...rest}
-              />
-            </Content>
+            {(!this.props.delayedContentRender || this.state.renderContent) && (
+              <Content manager={this._MrManager} id={this._portalWrapperId}>
```

Comment:
`MrManager` 😆

Reply:
Every time I look into PortalWrapper, there's this piece of code:
```
// Welcome aboard, Mr. Manager!
_MrManager = setupManager(managerNamespace)
// Wow, I'm Mr. Manager!
// Well, manager… we we just say manager.
```
:stuck_out_tongue: